onnx_mxnet/tests/test_layers.py

```diff
@@ -325,6 +325,32 @@ def test_max(self):
         output = mxnet_backend.run_node(node, [input1])[0]
         npt.assert_almost_equal(output, input1)
 
+    def test_deconv(self):
+        """ Test Deconvolution operator"""
+        input1 = np.arange(9).reshape((1, 1, 3, 3))
+        weight = np.ones(shape=(1, 3, 3, 3))
+
+        result = np.array([[[[0., 1., 3., 3., 2.],
```

Comment:
is there a specific reason of creating array specifying the values? Can't you use random array setting?

Reply:
If we use random input, there is no easy way to compare the result with a numpy operation result for complex operations like (Convolution and deconv)